TERMS.md

```diff
@@ -0,0 +1,97 @@
+- **Service**
+	- (General) A service is a software functionality or set of functionalities that can be activated by a client. A key characteristic of a service in support of SOA is that it has low coupling to other services and strives to provide a useful function also if used independently from other services.
+	- (VSC) A VSC **Service** contains one or more **Interface**s, which are defined or referenced within the service definition.  The service definition may within its namespace also define **Type**s, constants and other definitions required to make use of the service.
+	- A Service name is not in itself a Namespace, but its contents must be defined within *at least one* **Namespace**.  The Namespace in question is referenced or defined from within the Service definition.
+	- A Service must have its fundamental definition in **one** single .yaml formatted file, although this file might use references to other parts.  In other words, all Interfaces that make up a service must be mentioned within this file*.
+	- A file can contain a maximum of one Service.
+
+_>	(To be further determined - how complete is our include/reference concept?)_
+
+- **Namespace**
+	- A namespace is a way to separate definitions in named groups.
+	- Namespaces are used to ensure that local names will not clash with identically named items in other namespace.
+	- Namespaces usually also separate what objects can be *seen* or *reached* from within a part of a program.
+	- Namespaces can contain other Namespaces, creating a hierarchy.
+	- How visibility/reachability is handled *might* be specific to the target language or environment, but VSC generally assumes that items within different (non-parent) namespaces are isolated from each other unless otherwise specified, and that common hierarchy rules apply (i.e. everything within a child namespace can see and reach items in any of its parent namespaces without a specific required statement of reference or inclusion).
```

Comment:
Now which synonym do we prefer  :) @slawr

https://www.textfocus.net/en/synonyme/isolation